api/src/services/channel-service.ts

```diff
@@ -23,8 +24,21 @@ export class ChannelService {
 		this.password = config.statePassword;
 	}
 
-	create = async (identityId: string, topics: Topic[], encrypted: boolean, seed?: string): Promise<CreateChannelBodyResponse> => {
-		const res = await this.streamsService.create(seed);
+	async create(
```

Comment:
If you turn the parameters list into an object, the order of parameters won’t matter